pkg/kubelet/cm/dra/state/state_checkpoint_test.go

```diff
@@ -59,12 +59,37 @@ func TestCheckpointGetOrCreate(t *testing.T) {
 				Namespace:  "default",
 				PodUIDs:    sets.New("139cdb46-f989-4f17-9561-ca10cfb509a6"),
 				CdiDevices: []string{"example.com/example=cdi-example"},
+				ClassName:  "class-name",
 			},
 			},
 		},
+		{
+			"Restore valid checkpoint with 2 claims",
```

Comment:
Looking again, this LGTM (but I haven't explored deeply yet) but does it belong in this PR? Meaning, shouldn't it be part of the PR which adds checkpoint/state support to DRA?

Reply:
I'm confused by the inclusion of this in this PR as well. I have already added similar tests in https://github.com/kubernetes/kubernetes/pull/116513, which this PR should be rebased on top of.